packages/seacas/libraries/ioss/src/unit_tests/UnitTestReadAllBlockData.C

```diff
@@ -0,0 +1,399 @@
+// Copyright(C) 1999-2020, 2022 National Technology & Engineering Solutions
+// of Sandia, LLC (NTESS).  Under the terms of Contract DE-NA0003525 with
+// NTESS, the U.S. Government retains certain rights in this software.
+//
+// See packages/seacas/LICENSE for details
+
+#include <string>
+#include <vector>
+
+#include <sys/time.h>  // for gettimeofday, timeval, timezone
+
+#ifdef SEACAS_HAVE_MPI
+#include "mpi.h"
+#endif
+#include "gtest/gtest.h"
+
+#include <fstream>
+#include <random>
+#include <algorithm>
+#include <iostream>
+#include <unistd.h>                     // for unlink
+
+#include <Ionit_Initializer.h>
+#include <Ioss_ParallelUtils.h>
+#include <Ioss_IOFactory.h>
+#include <Ioss_DBUsage.h>
+#include <Ioss_DatabaseIO.h> // for DatabaseIO
+#include <Ioss_Field.h>          // for Field, etc
+#include <Ioss_Property.h>
+
+#include <Ioss_NodeBlock.h>
+#include <Ioss_ElementBlock.h>
+#include <Ioss_ParallelUtils.h>
+#include <Ioss_Region.h>
+
+namespace {
+double wall_time()
```

Comment:
There is an `Ioss::Utils::timer()` which can be used and eliminates the need for including `sys/time.h` which doesn't exist on windows, so need ifdefs (See libraries/supes/ext_lib/excpus.c for example)

Reply:
I was wondering about that ... good to know